commands/music/music-trivia.js

```diff
@@ -200,20 +208,13 @@ module.exports = class MusicTriviaCommand extends Command {
                   1
               );
               msg.react('☑');
-            } else if (
-              msg.content.toLowerCase() ===
-                queue[0].singer.toLowerCase() +
-                  ' ' +
-                  queue[0].title.toLowerCase() ||
-              msg.content.toLowerCase() ===
-                queue[0].title.toLowerCase() +
-                  ' ' +
-                  queue[0].singer.toLowerCase()
-            ) {
-              if (
-                (songSingerFound && !songNameFound) ||
-                (songNameFound && !songSingerFound)
-              ) {
+            } 
```

Comment:
Can you explain lines 212-217? Line 212 says 'if user guessed song name' but line 217 checks for both name and singer

Reply:
This is the same logic as before. The if clause on 213 checks if the _user's guess_ is correct for the song name. The if clause on 214 check's if the song name had already been previously guessed or not. The if clause on 217 tests to see if both things have been guessed now and if it should move into the next song.